Sources/Private/Experimental/Layers/BaseCompositionLayer.swift

```diff
@@ -35,46 +35,23 @@ class BaseCompositionLayer: CALayer, AnimationLayer {
 
   // MARK: Internal
 
-  /// The layer that the `LayerModel.transform` is applied to
-  ///  - Child `LayerModel`s should always be added to this layer,
-  ///    so they inherit the transform of the parent layer.
-  let transformLayer: CALayer = {
-    let layer = CALayer()
-    layer.name = "transform"
-    return layer
-  }()
-
-  /// The layer that should render visible elements displayed by this layer's `LayerModel`
-  ///  - This layer applies `LayerModel.transform.opacity`, which applies to this specific
-  ///    `LayerModel`'s content and not the content of any child `LayerModel`s.
-  let contentsLayer: CALayer = {
```

Comment:
The distinction between `transformLayer` and `contentsLayer` is no longer necessary, now that we don't add child `LayerModel`s as sublayers